### ISSUE TITLE
Improve landing page for local hosted page

### ISSUE THREAD

**Issue by sebastinez** [radicle-dev/radicle-interface#243]
If we want to allow people to mount the web client on their machines we should improve the heuristic for getting a Web3 Provider.
Else they would be welcomed with the following, which won't give the end user much to work with.
<img width="462" alt="Bildschirmfoto 2022-06-22 um 08 58 32" src="https://user-images.githubusercontent.com/7912302/174964269-80226924-6f2d-4898-95b6-b3ce72bc4707.png">

I'm thinking we could:
1. Improve the error message when hosted on `localhost` or outside of `app.radicle.network`.
2. Use a ethers.fallbackProvider with some alert on the loaded page that the user should provide an Alchemy API Key.

**Comment by brandonhaslegs** [comment]
Came here to report this. 👆

**Comment by cloudhead** [comment]
Yeah even just adding "A wallet such as Metamask must be enabled on your browser." would help
